examples/kafka-broker/verify.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+export NAME=kafka-broker
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Waiting for Zookeeper and Kafka started."
+sleep 40
```

Comment:
Long time to wait, could we use a loop to check them are ready?

Reply:
updated, please help review again